src/doc/src/reference/build-scripts.md

```diff
@@ -74,16 +74,23 @@ directory specified in the [`OUT_DIR` environment variable][build-env]. Scripts
 should not modify any files outside of that directory.
 
 Build scripts communicate with Cargo by printing to stdout. Cargo will
-interpret each line that starts with `cargo:` as an instruction that will
+interpret each line that starts with `cargo::` as an instruction that will
 influence compilation of the package. All other lines are ignored.
 
-> Note: The order of `cargo:` instructions printed by the build script *may*
+> **Note:** The old invocation prefix `cargo:` (one colon only) is deprecated
+> and won't get any new features. To migrate, use two-colons prefix `cargo::`,
+> which was added in Rust 1.75. If you were using `cargo:KEY=VALUE` for
```

Comment:
I believe this needs to be the same version as this PR, correct?

```suggestion
> which was added in Rust 1.76. If you were using `cargo:KEY=VALUE` for
```

Reply:
Updated. Thanks!